interface/src/app/main/Help.tsx

```diff
@@ -54,43 +62,70 @@ const Help = () => {
-              <ListItemAvatar>
-                <Avatar sx={{ bgcolor: '#72caf9' }}>
-                  <CommentIcon />
-                </Avatar>
-              </ListItemAvatar>
-              <ListItemText primary={LL.HELP_INFORMATION_2()} />
-            </ListItemButton>
-          </ListItem>
-
-          <ListItem>
-            <ListItemButton
-              component="a"
-              href="https://github.com/emsesp/EMS-ESP32/issues/new/choose"
-            >
-              <ListItemAvatar>
-                <Avatar sx={{ bgcolor: '#72caf9' }}>
-                  <GitHubIcon />
-                </Avatar>
-              </ListItemAvatar>
-              <ListItemText primary={LL.HELP_INFORMATION_3()} />
-            </ListItemButton>
-          </ListItem>
-        </List>
+        {me.admin ? (
+          <List sx={{ borderRadius: 3, border: '2px solid grey' }}>
+            <ListItem>
+              <ListItemButton component="a" href="https://emsesp.org">
+                <ListItemAvatar>
+                  <Avatar sx={{ bgcolor: '#72caf9' }}>
+                    <MenuBookIcon />
+                  </Avatar>
+                </ListItemAvatar>
+                <ListItemText primary={LL.HELP_INFORMATION_1()} />
+              </ListItemButton>
+            </ListItem>
+
+            <ListItem>
+              <ListItemButton component="a" href="https://discord.gg/3J3GgnzpyT">
+                <ListItemAvatar>
+                  <Avatar sx={{ bgcolor: '#72caf9' }}>
+                    <CommentIcon />
+                  </Avatar>
+                </ListItemAvatar>
+                <ListItemText primary={LL.HELP_INFORMATION_2()} />
+              </ListItemButton>
+            </ListItem>
+
+            <ListItem>
+              <ListItemButton
+                component="a"
+                href="https://github.com/emsesp/EMS-ESP32/issues/new/choose"
+              >
+                <ListItemAvatar>
+                  <Avatar sx={{ bgcolor: '#72caf9' }}>
+                    <GitHubIcon />
+                  </Avatar>
+                </ListItemAvatar>
+                <ListItemText primary={LL.HELP_INFORMATION_3()} />
+              </ListItemButton>
+            </ListItem>
+          </List>
+        ) : (
+          <Stack
+            spacing={1}
+            padding={1}
+            direction="row"
+            divider={<Divider orientation="vertical" flexItem />}
+            sx={{
+              borderRadius: 3,
+              border: '2px solid grey',
+              justifyContent: 'space-around',
+              alignItems: 'center'
+            }}
+          >
+            <Typography border="red" variant="subtitle1">
+              {LL.HELP_INFORMATION_5()}
+            </Typography>
+            <Box
+              padding={1}
+              component="img"
+              sx={{
+                maxHeight: { xs: 100, md: 250 }
+              }}
+              src="https://emsesp.org/_media/images/installer.jpeg"
```

Comment:
Funny image,
I'm missing a ["CEE-to-gardena"](https://veranstaltungstechnik-event.de/wp-content/uploads/2019/03/Gardena2CEE-1.jpg) to make it perfect.

Reply:
haha! I can add it that interface! I made the image in like 30 seconds by going to https://designer.microsoft.com/image-creator and typing in "installer boiler". This was the first image that came out. The nice thing about ChatGPT/AI is that the images created are not copyrighted and can be used everywhere. I'm still waiting for the day when Bosch will shut us down :-(